react-props-and-expressions/this-app/src/CustomButton.jsx

```diff
@@ -0,0 +1,3 @@
+export default function CustomButton({ props }) {
+  return <button style={props.color}>{props.text}</button>;
+}
```

Comment:
The property names you're using in the `CustomButton` definition don't match the names it's used with in `App.jsx`.